src/types.ts

```diff
@@ -59,13 +59,25 @@ export type SendMessageBrowserOptions = {
   abortSignal?: AbortSignal
 }
 
+interface CreateChatCompletionStreamResponse
+  extends openai.CreateChatCompletionDeltaResponse {
+  usage: CreateCompletionStreamResponseUsage
+}
+
+interface CreateCompletionStreamResponseUsage
+  extends openai.CreateCompletionResponseUsage {
+  estimated: true
+}
+
 export interface ChatMessage {
   id: string
   text: string
   role: Role
   name?: string
   delta?: string
-  detail?: any
+  detail?:
```

Comment:
It actually should not be nullable. In the current logic it can never be undefined in final message object. But I could not find a good way how to type it correctly with current mutation-like codestyle

Reply:
it's fine to leave it nullable for now. appreciate the note